recipes/zlib/1.2.11/patches/0001-no-exampels-cmake.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 0fe939d..1e3a601 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -226,24 +226,3 @@ if(NOT SKIP_INSTALL_FILES AND NOT SKIP_INSTALL_ALL )
+     install(FILES ${ZLIB_PC} DESTINATION "${INSTALL_PKGCONFIG_DIR}")
+ endif()
+ 
+-#============================================================================
```

Comment:
shouldn't be needed, as we build only particular target right now

Reply:
you mean the patch at all? 
if, than it is since binaries need to be signed when you build them with xcode for iOS
if you mean just the line about the comment, than will stay unchagned

Reply:
about the whole patch. we're not running `make all`, I think we're running `make zlib` (or `make zlibstatic`), thus we don't build all the targets (tests, examples, etc.), but only the library itself.

Reply:
yes, that is true
I will revert it (even if my personal opinion is, as long as these lines are not in a enable_exampe/testing condition, they are a bug ;-) )